test/default-messages.js

```diff
@@ -1,37 +1,32 @@
 var tap = require('tap');
 var spawn = require('child_process').spawn;
-var trim = require('string.prototype.trim');
+var concat = require('concat-stream');
 
 tap.test('default messages', function (t) {
     t.plan(1);
 
-    var tc = tap.createConsumer();
+    var ps = spawn(process.execPath, [ __dirname + '/messages/defaults.js' ]);
```

Comment:
_And_ if we change this _one_ instance, should we change _all_ (**Four**) occurrences: 
https://github.com/substack/tape/search?utf8=%E2%9C%93&q=spawn%28process.execPath ? (_just say the word and I'll update them all to be consistently windows-friendly_)